src/iop/Permutohedral.h

```diff
@@ -449,7 +452,7 @@ template <int D, int VD> class PermutohedralLattice
   }
 
   /* Merge the multiple threads' hash tables into the totals. */
-  void merge_splat_threads(void)
+  void merge_splat_threads() const
```

Comment:
Member functions should be const unless they modify the object's internal state or leak a non-const pointer or reference to data members.

Reply:
Note that `merge_splat_threads` **does** modify internal state (it combines the contents of all of the per-thread hash tables into the hash table for thread number 0).  The code still compiles with the `const` only because the modified state is in a dynamic allocation instead of a direct member of the object.

Reply:
I will remove the `const` here.